atomicapp/nulecule/lib.py

```diff
@@ -1,11 +1,18 @@
 # -*- coding: utf-8 -*-
+import sys
+
+import logging
+
```

Comment:
We can have a new line between stdlib imports and atomicapp, for readability :+1: